### DISCUSSION TITLE
Welcome to go-libs Discussions!

### DISCUSSION THREAD

**Discussion by Mahes2** [Mahes2/go-libs#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kaamkiya** [comment]
Hi! I read your comment on DEV.to about this repo. Do the modules I add have to be useful?

My current idea is something that uses ANSI escape sequences to print to the terminal with color. I can't think of any practical application for this, but people might use it for warnings. 

Also, would I still put it in the `data_struct` directory?

Thanks!

**Reply by Mahes2** [reply]
Hi @Kaamkiya , actually i'm open to any utilities. But it would be great if you can show me some result by screenshot it first?
Thanks !

**Reply by Kaamkiya** [reply]
Sure! I'll finish it and, send a screenshot, and then make a PR :)
